docs/advanced/full-node.md

```diff
@@ -0,0 +1,63 @@
+# <img class="dcr-icon" src="/img/dcr-icons/AtoB.svg" /> Running a Full Node
+
+---
+
+This guide explains everything you need to know about running a full node on Decred.
```

Comment:
"running a Decred full node"?

Reply:
Was on the fence on that myself. Changed to 'Decred full node', as I think that is probably the convention actually.